Cron/PendingOrdersUpdate.php

```diff
@@ -74,17 +68,18 @@ public function execute(): static
 
     protected function getPendingOrders(): array
     {
-        $currentTime = new PhpDateTime();
-        $fromTime = (clone $currentTime)->sub(new \DateInterval('PT2H')); // 2 hours ago
-        $toTime = (clone $currentTime)->sub(new \DateInterval('PT1H'));   // 1 hour ago
+        date_default_timezone_set($this->config->getTimezone());
```

Comment:
Are we sure this is needed? Generally the timestamp which is stored in the DB would be using the correct datetime zone automatically as it should be set as part of the server config